Tests/HarborTests/Mocks/MocksRequest.swift

```diff
@@ -7,7 +7,7 @@
 
 import Harbor
 
-final class MockGetRequestService: HGetRequestProtocol {
+final class MockGetRequestService: HGetRequestProtocol, @unchecked Sendable {
```

Comment:
**@unchecked Sendable** is that ok? Is there a way to avoid the declaration of this in the requests?

Reply:
Its ok because we are using a class instead of a struct, generally we should use structs for requests, like in `RESTRequest`, this is cause we are testing